### PR TITLE
https://developers.google.com/ml-kit/migration

### PR DESCRIPTION
**Pull request by radvansky-tomas**
Plugin is using outdated lib...this PR should resolve those issues

### CODE REVIEW

**Comment by bounty1342** [comment]
Updating Crashlytics gave me error. Had to import on build.graddle several dependencies to solve it...
If updated, I should not need them anymore.

**Comment by rmtmckenzie** [comment]
Oh, missed this PR. I'll try to get this integrated soon - not depending on firebase would also be great!

File: android/src/main/java/com/github/rmtmckenzie/qrmobilevision/QrCameraC2.java [hunk 1]
```diff
@@ -212,8 +211,8 @@ private Integer afMode(CameraCharacteristics cameraCharacteristics) {
         }
 
         @Override
-        public FirebaseVisionImage toImage() {
-            return FirebaseVisionImage.fromMediaImage(image, firebaseOrientation);
+        public InputImage toImage() {
+            return InputImage.fromMediaImage(image, firebaseOrientation);
```
Comment (howardwkim) [inline android/src/main/java/com/github/rmtmckenzie/qrmobilevision/QrCameraC2.java:215]: It appears that `InputImage` only accepts 0, 90, 180, 270 for the second parameter (rotationDegrees). 
`getFirebaseOrientation` returns 0, 1, 2, 3.

Comment (howardwkim) [inline android/src/main/java/com/github/rmtmckenzie/qrmobilevision/QrCameraC2.java:215]: It appears that `InputImage` only accepts 0, 90, 180, 270 for the second parameter (rotationDegrees).
`getFirebaseOrientation` returns 0, 1, 2, 3.

**Pull request merged**